config/example.toml

```diff
@@ -57,7 +57,7 @@ https = false
 type = "postgresql"
 
 # Database.host - The ip address of the machine hosting the database.
-host = "127.0.0.1"
+host = "db"
```

Comment:
Could we potentially expose the db host via an environment variable. That way the user can easily specify the location of the database through the `docker-compose.yml`

Reply:
Exposing the environment variable like that would add extra steps for non-Docker installs, but I am certainly open to it.

Would like to see what others think as well. :)

Reply:
Is there a way we could have both? So if the environment variable is not set then it defaults to what's in the config file?

Reply:
yeah probably?
I am asking about this in the chat room :)